src/reactviews/pages/TableExplorer/TableDataGrid.tsx

```diff
@@ -96,6 +96,23 @@ export const TableDataGrid = forwardRef<TableDataGridRef, TableDataGridProps>(
         function reactGridReady(reactGrid: SlickgridReactInstance) {
             reactGridRef.current = reactGrid;
             isInitializedRef.current = true;
+
+            // Commit any active edits when the grid loses focus
+            if (reactGrid.slickGrid) {
+                const gridElement = reactGrid.slickGrid.getContainerNode();
+                if (gridElement) {
+                    gridElement.addEventListener("focusout", (event: FocusEvent) => {
+                        // Check if focus is leaving the grid container entirely
+                        const relatedTarget = event.relatedTarget as HTMLElement;
+                        if (!relatedTarget || !gridElement.contains(relatedTarget)) {
+                            // Commit the current edit if any
+                            if (reactGrid.slickGrid?.getEditorLock().isActive()) {
+                                reactGrid.slickGrid.getEditorLock().commitCurrentEdit();
```

Comment:
Where is this committing the changes to?  Can you add that in the comment?  I'm assuming (hoping) not all the way to the database.